doc/sphinx/language/gallina-extensions.rst

```diff
@@ -1011,7 +985,7 @@ Reserved commands inside an interactive module type:
 
    This is a shortcut for the command :n:`Include @module` for each :token:`module`.
 
-.. cmd:: @assumption_keyword Inline @assums
+.. cmd:: @assumption_token Inline @assums
```

Comment:
Actually, IIRC I touched this only because assumption_token was renamed and Sphinx would complain about the missing symbol.

The explanation doesn't say much to me, not does it cover the correct syntax for `Inline` that I have in the gallina chapter.  Nor do I see where a ! annotation fits in unless it is referring to an `Include`.  If you want to give me a good description, I'll drop it in and remove the text here.

```
The instance of this assumption will be automatically expanded at functor application,
except when this functor application is prefixed by a ! annotation.
```

![image](https://user-images.githubusercontent.com/1253341/74581673-c34bba80-4f66-11ea-87a8-0f7e29e0b1ca.png)

Reply:
If you want to make this 2 commands (or a command variant) that's OK with me.  Just hope we use that mechanism sparingly so it's easy to find all the possible options.

Reply:
Unfortunately, I didn't know this syntax and have yet to look at this part of the doc.